lib/oregon_digital/controlled_vocabularies/media_type.rb

```diff
@@ -0,0 +1,14 @@
+module OregonDigital
+  module ControlledVocabularies
+    class MediaType < ActiveTriples::Resource
+      configure rdf_label: ::RDF::Vocab::DC.format
```

Comment:
This isn't right. I'm not positive you even need to define `rdf_label` at all here, it's an ActiveTriples::Resource already. Plus you're defining the label as the predicate URI, which is circular.

Reply:
I did this because thats how location is done. https://github.com/samvera/hyrax/blob/5a9d1be16ee1a9150646384471992b03aab527a5/lib/hyrax/controlled_vocabularies/location.rb#L4 The RDF label I think is still able to be expressed as an ntriple which wants a predicate. But if thats not the case, then I can explore not configuring a specific predicate for an rdf_label for a specific controlled vocabulary